packages/api/src/services/library_item.ts

```diff
@@ -700,10 +700,16 @@ export const findRecentLibraryItems = async (
 }
 
 export const findLibraryItemsByIds = async (ids: string[], userId: string) => {
+  const selectColumns = getColumns(libraryItemRepository)
```

Comment:
Just wondering, would this get called by search or get article APIs?

Reply:
No, it only gets called here